views/calendar.twig

```diff
@@ -1 +1,34 @@
-{{ message }}
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="Technology,Lancaster,Coding,Programming,Meetup">
```

Comment:
i'm pretty sure that came from [you](https://github.com/TechLancaster/tl-website/commit/17b3e62b45118348978e8feefc729c51ef859f0c), hahaha